pyhomematic/devicetypes/actors.py

```diff
@@ -594,15 +594,27 @@ def __init__(self, device_description, proxy, resolveparamsets=False):
         # init metadata
         self.SENSORNODE.update({"DOOR_STATE": [1]})
 
-    def move_up(self):
+    def is_closed(self):
+        """Returns whether the door is closed"""
+        state = self.getValue("DOOR_STATE", channel=1)
+        # States:
+        # 0: closed
+        # 1: open
+        # 2: ventilation
+        # 3: unknown
+        if state in [2, 3]:
+            return None
+        return state == 0
+
+    def move_up(self, channel=1):
         """Opens the garage"""
         return self.setValue("DOOR_COMMAND", 1, channel=1)
```

Comment:
Shouldn't the be `channel=channel` for the channel-argument to `move_up` to have any function?

Reply:
I didn't want to change the channel assignment, since the hard-coded 1 works. OTOH, I assumme that the entity in home assistant will have the right channel anyway, so I can change that.

Reply:
I think there's a misunderstanding. The `setValue` call will always use `1`. The channel-definition in `mode_up` isn't being used. In my opinion it should look like this:

```python
def move_up(self, channel=1):
    """Opens the garage"""
    return self.setValue("DOOR_COMMAND", 1, channel=channel)
```

Reply:
That's what I understood from your first comment as well. I tried that, but it doesn't work; I got an error because somehow the commands where being sent on channel 2 :-(

I renamed the parameter to "_", because it is unused, but needed for home assistant to be able to call the methods.

Reply:
Well, in that case better leave it as `channel=1`. Using a `_` there is not a good workaround.

Reply:
Done. I have also added comments to make it clear why the parameter is being ignored.